apps/i18n/netsim/en_us.json

```diff
@@ -2,6 +2,8 @@
   "a_and_b": "A/B",
   "addPacket": "Add Packet",
   "ascii": "ASCII",
+  "autoDnsNotFound": "NOT_FOUND",
```

Comment:
Hmm... that's fair.  This response is part of the auto-DNS protocol, and perhaps should be left untranslated because of that?  I don't want any spaces in this string so that it conforms to the response pattern, even though we don't expect these responses to be machine-parsed.